extensions/sql/runtime/src/main/doc/limitations.adoc

```diff
@@ -0,0 +1,4 @@
+While you can use any of the available database types in JVM mode, Oracle database type negatively affects serialization in native mode.
+See quarkus issue https://github.com/quarkusio/quarkus/issues/20396[#https://github.com/quarkusio/quarkus/issues/20396] for more details.
+
+https://camel.apache.org/components/next/sql-component.html#_using_the_jdbc_based_aggregation_repository[Aggregation repository] does not work in native mode with Oracle db, because of this issue.
```

Comment:
```suggestion
xref:{cq-camel-components}::sql-component.adoc#_using_the_jdbc_based_aggregation_repository[Aggregation repository] does not work in native mode with Oracle DB, because of this issue.
```

Reply:
done